.github/workflows/build_and_test.yml

```diff
@@ -193,16 +193,15 @@ jobs:
         with:
           submodules: true
 
-#      Disabled for now
-#      - name: Restore cargo deps cache
-#        id: cache_cargo
-#        uses: actions/cache@v4
-#        with:
-#          path: |
-#            !~/.cargo/registry/src
-#            ~/.cargo/git/
-#            target/
-#          key: v1-${{ runner.os }}-${{ runner.arch }}-cargo-clippy-${{ hashFiles('rust-toolchain.toml') }}-${{ hashFiles('Cargo.lock') }}
+      - name: Restore cargo deps cache
+        id: cache_cargo
```

Comment:
I don't think we need `id` here

Reply:
Also in `neon_extra_buiild` we use a bit different cache-key, probably it makes sense to unify it:
https://github.com/neondatabase/neon/blob/264c34dfb7b90e619677fc04fdb957b270c40e8e/.github/workflows/neon_extra_builds.yml#L112C1-L120C140

Reply:
Awesome. I've unified them.